packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -459,7 +460,7 @@ export class LedgerKeyring extends EventEmitter {
     return signature;
   }
 
-  async unlockAccountByAddress(address: string): Promise<string | undefined> {
+  async unlockAccountByAddress(address: Hex): Promise<string | undefined> {
     const checksummedAddress = ethUtil.toChecksumAddress(address);
```

Comment:
Maybe we could use `getChecksumAddress` instead? (similarly to the Trezor's PR)

Reply:
Good idea. Done in [6b04afc](https://github.com/MetaMask/accounts/pull/194/commits/6b04afc00591c9ef35d2403940963c27d69845e2)